crates/release_plz_core/tests/all/git_only.rs

```diff
@@ -0,0 +1,175 @@
+use crate::helpers::git_only_test::{GitOnlyTestContext, PROJECT_NAME};
+use cargo_metadata::semver::Version;
+use release_plz_core::PackagesUpdate;
+use std::path::PathBuf;
+
+fn touch(path: impl Into<PathBuf>) -> std::io::Result<()> {
+    fs_err::OpenOptions::new()
+        .create(true)
+        .write(true)
+        .open(path)
+        .map(|_| ())
+}
```

Comment:
cool function! can you use it in your changes to `crates/release_plz/tests/all/release_pr.rs` as well?